mypy.ini

```diff
@@ -8,3 +8,7 @@ mypy_path = stubs
 [mypy-tests]
 check_untyped_defs = True
 disallow_untyped_defs = False
+
+[mypy-third_party.*]
+ignore_errors = True
+ignore_missing_imports = True
```

Comment:
This was supposed to help for linting issue when importing the module but it didn't so I had to add ` # type: ignore` there anyway.